src/CUDAdrv.jl

```diff
@@ -3,21 +3,24 @@ __precompile__()
 module CUDAdrv
 
 using Compat
-import Compat.String
+using Compat.String
```

Comment:
Why this change? The [Compat.jl README](https://github.com/JuliaLang/Compat.jl#usage) suggests otherwise.